app/frontend/src/store/modules/user.js

```diff
@@ -14,6 +14,22 @@ export default {
       if (Array.isArray(acronyms)) {
         state.acronyms = acronyms;
       }
+    },
+    setAcronymClientStatus: (state, clientsForUser) => {
+      if (Array.isArray(clientsForUser) && clientsForUser.length) {
+        state.acronyms.forEach(acr => {
+          const clientSet = clientsForUser.find(client => client.acronym === acr.acronym);
+          acr.devStatus = (clientSet && clientSet.dev && clientSet.dev.enabled) ? true : false;
+          acr.testStatus = (clientSet && clientSet.test && clientSet.test.enabled) ? true : false;
+          acr.prodStatus = (clientSet && clientSet.prod && clientSet.prod.enabled) ? true : false;
```

Comment:
All of these lines don't need a ternary - you can just do the following and it'll be the same:
`acr.devStatus = clientSet && clientSet.dev && clientSet.dev.enabled`

Reply:
Yes right, had left that in from when I was doing text.